C-Plus-Plus/ds/Priority_Queue_using_Linked_List.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+Program by :: Anand Dasani 
+Date :: 17-March-2021
+AIM :: To implement Priority Queue (PQ) using Linked List (LL).
+
+                                ALGO
+1. Take  data from the user 
+2. Take priority of that data from the user 
+3. According to priority traverse the linked list 
+    (a) if linked list is empty then insert the node at the beginning 
+    (b) else insert that data at it's appropriate place 
+4. Deletion will be done by front only 
+    (a) if linked list is empty then so error message "PQ UNDERFLOW"
+    (b) else delete the first node everytime user want the deq 
+5. Exit 
```

Comment:
Remove this

Reply:
@HarshCasper sir from next time onwards should i write algorithm or not ?